proposals/notifications.md

```diff
@@ -0,0 +1,88 @@
+# Solid Notifications
+
+Solid notifications are intended to announce social interactions and data. A Solid notification is an [LDP Resource](https://www.w3.org/TR/ldp/#ldpr) which is submitted to and resides in an [LDP Container](https://www.w3.org/TR/ldp/#ldpc), i.e., a [Solid inbox](inbox.md).
+
+The contents of a notification is application specific. We derive two core principles from this:
+
+1. a notification can contain arbitrary data
+2. a notification is vocabulary agnostic
```

Comment:
Mentioned `solid:Notification` as a 'should' for LDP RDF Sources. It makes sense to include them in the notifications in order to handle different cases i.e., the notification resource may not necessarily reside as a "file" on the system, but may be in an RDF store for instance. So, if the data gets moved around, at least the context moves around with it. Otherwise, `solid:Notification` is not a 'must' for the purpose of discovery.